content/docs/introduction/getting_started.md

```diff
@@ -186,16 +169,13 @@ section in your `prometheus.yml` and restart your Prometheus instance:
 
 ```
 scrape_configs:
-  - job_name:       'example-random'
-
-    # Override the global default and scrape targets from this job every 5 seconds.
+  - job_name: 'example-random'
+    # Scrape targets from this job every 5 seconds.
     scrape_interval: 5s
```

Comment:
Maybe kill the 5s so people see that the time interval for their data is different.